src/SignalR/clients/ts/signalr-protocol-msgpack/src/Utils.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+/** @private */
+export function isArrayBuffer(val: any): val is ArrayBuffer {
```

Comment:
Same code exists in 3.1, looks like updating the buffer version causes `instanceof` to not work anymore.